modules/govuk_unattended_reboot/templates/usr/local/bin/with_reboot_lock.erb

```diff
@@ -2,7 +2,8 @@
 
 set -eu
 
-lock_file="<%= @lock_dir %>/reboot_lock"
+lock_file_name="$1"
+lock_file="<%= @lock_dir %>/$lock_file_name"
```

Comment:
I'd quite like to have the file still have the same prefix, just in case we need to find these lock files to unlock later if something's become gummed up

```suggestion
lock_file="<%= @lock_dir %>/reboot_lock-${lock_file_name}"
```

Reply:
That sounds fine to me.